### ISSUE TITLE
gfycat not displaying

### ISSUE THREAD

**Issue by ghost** [aledeg/FreshRSS-extensions#2]
Hello

Sometime video are not displayed at all
https://zippy.gfycat.com/ElectricClosedIndochinesetiger.mp4

the video player is displayed but the video doesn't start.
you can try with this flux : https://www.reddit.com/r/Celebhub/.rss

**Comment by aledeg** [comment]
Do you have more info on the video? I cannot find it on the feed.

**Comment by ghost** [comment]
this one for example
nsfw by the way
https://www.reddit.com/r/celebnsfw/comments/cbnfrl/marisa_tomei_pierced_nipples/

**Comment by aledeg** [comment]
I've just added the feed to my reader. I do not have that article in it.
Could you paste the content of ```<div class="reddit-image figure">``` in here. So I can see what was generated.

**Comment by ghost** [comment]
`<video controls="" class="reddit-image"><source src="https://giant.gfycat.com/CarefulWildFunnelweaverspider.mp4" type="video/mp4"></video><p class="caption"><a href="https://www.reddit.com/r/celebnsfw/comments/cbnfrl/marisa_tomei_pierced_nipples/">Comments</a></p>`

**Comment by aledeg** [comment]
Thank you, I will have a look at it!

**Comment by aledeg** [comment]
I cannot reproduce it but I know that sometimes, it doesn't work. When this happens, usually it's because of the case. It is different between the content of the RSS feed and the actual video. So far, I didn't find a good way to make it work.

**Comment by ghost** [comment]
Thanks for the time you have spent.

**Comment by aledeg** [comment]
@prysme01 I figured why it's not working but I still don't know how to fix that.

The reason behind that behavior is that the number of calls to gfycat is too important and some are failing. Thus you have an empty video player. As I aforementioned, I don't know how to fix that.

But I've found a workaround that behavior. If you limit the number of entries displayed to `20` and you disable `Load more articles at the page bottom` in the reading configuration page, it looks like it's working properly.
